### DISCUSSION TITLE
123盘 Mac 系统无法解析123盘视频

### DISCUSSION THREAD

**Discussion by YJLAugus** [AlistGo/alist#755] [Q&A] [answered]
### Alist Version / Alist 版本

v2.3

### Describe the bug / 问题描述

视频无法在线观看，403 禁止

### Reproduction / 复现链接

nope

### Logs / 日志

```shell
nope
```

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by xhofe** [comment, marked as answer]
123pan有IP检查，请确定：
- 未使用代理工具
- 非局域网部署

**Reply by YJLAugus** [reply]
是的 局域网部署的，感谢解答
